inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/eltwise/eltwise_kernel_b_fs_yx_fsv16.cpp

```diff
@@ -68,7 +71,7 @@ JitConstants EltwiseKernel_b_fs_yx_fsv16::MakeLoadJitConstants(const eltwise_par
                     if (params.inputs[input.index].LogicalSize() == params.output.Feature().v &&
                         params.inputs[input.index].LogicalSize() == params.inputs[input.index].Feature().v) {
                         jit.AddConstant(MakeJitConstant(name,
-                                                        "UNIT_BLOCK_READ(input" + std::to_string(input.index) +
+                                                        "DT_INPUT_BLOCK_READ(input" + std::to_string(input.index) +
```

Comment:
It can't be assumed that all inputs have same type. `DT_INPUT_BLOCK_READ` is an alias for `BLOCK_WRITEN(INPUT0_TYPE, 1, ptr, offset, val)`. Macro with proper `INPUT<input_idx>_TYPE` should be used here.